app/forms/support_interface/multiple_provider_users_wizard.rb

```diff
@@ -0,0 +1,161 @@
+module SupportInterface
+  class MultipleProviderUsersWizard
```

Comment:
nit: Could we use meaningful variable names instead of letters in this class when iterating to help with  clarity? example: `provider_user` instead of `pu` etc.

Reply:
Ah sorry - I'm gone with the convention of using letters as var names within blocks. I can change if this. Wonder if there is a custom rubocop cop rule we need for this.

Reply:
Fix - https://github.com/DFE-Digital/apply-for-teacher-training/pull/4896/commits/d9915e8c7d1c9b369103095ca67b797546ff944d